feature-libs/customer-ticketing/root/config/default-customer-ticketing-config.ts

```diff
@@ -3,7 +3,7 @@ import { CustomerTicketingConfig } from './customer-ticketing-config';
 export const defaultCustomerTicketingConfig: CustomerTicketingConfig = {
   customerTicketing: {
     attachmentValidity: {
```

Comment:
Validity or Validation?  or moreover: attachmentRestrictions

Reply:
yes that's better